Extensions/xkit_preferences.js

```diff
@@ -614,6 +614,11 @@ XKit.extensions.xkit_preferences = new Object({
 		$("#xkit-control-panel-shadow").click(function() {
 			XKit.extensions.xkit_preferences.close();
 		});
+		$(document).keydown(function(e) {
```

Comment:
Totally missed the way `#xkit-control-panel` affects run. Thanks, I'll get on these.